gcloud/datastore/test_connection.py

```diff
@@ -14,6 +14,11 @@
 
 import unittest
 
+try:
+    from gcloud.datastore.connection import _HAVE_GRPC
+except ImportError:  # pragma: NO COVER
+    _HAVE_GRPC = False
```

Comment:
The conditional import should be unnecessary:  that name is present whether or not the gRPC stuff can be imported.